src/main/kotlin/net/atos/zac/app/documentcreation/DocumentCreationRestService.kt

```diff
@@ -65,6 +68,11 @@ class DocumentCreationRestService @Inject constructor(
     ): RestDocumentCreationAttendedResponse =
         zrcClientService.readZaak(restDocumentCreationAttendedData.zaakUuid).also {
             assertPolicy(policyService.readZaakRechten(it).creeerenDocument)
+            restDocumentCreationAttendedData.taskId?.let {
+                val task = flowableTaskService.findOpenTask(it)
+                    ?: throw TaskNotFoundException("No open task found with task id: '$it'")
+                assertPolicy(policyService.readTaakRechten(task).toevoegenDocument)
```

Comment:
I don't understand this line. Why assert the `toevoegenDocument` policy here? This code is about document creation right? Not adding a document? Should it not be `creeerenDocument` here instead of `toevoegenDocument`?

Reply:
Yeah it should. Changing this

Reply:
Changed to `creeerenDocument`